# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/stressapptest/hotfix-mips.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- stressapptest-1.0.11/src/os.h.vanilla	2024-07-28 14:21:05.904901961 +0200
+++ stressapptest-1.0.11/src/os.h	2024-07-28 14:22:25.788900997 +0200
@@ -288,7 +288,7 @@
     tsc = (static_cast<uint64>(tbu) << 32) | static_cast<uint64>(tbl);
 #elif defined(STRESSAPPTEST_CPU_X86_64) || defined(STRESSAPPTEST_CPU_I686)
     tsc = __rdtsc();
-#elif defined(STRESSAPPTEST_CPU_MIPS)
+#elif defined(STRESSAPPTEST_CPU_MIPS_OCTEON)
     __asm __volatile("rdhwr  %0, $2\n" : "=r" (tsc));
 #elif defined(STRESSAPPTEST_CPU_ARMV7A)
     #warning "Unsupported CPU type ARMV7A: your timer may not function correctly"
